dependent/herzschrittmachertherapie-elektrophysiologie.csl

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer-fachzeitschriften-medizin -->
   <info>
-    <title>Herzschrittmachertherapie + Elektrophysiologie</title>
-    <title-short>Herzschr Elektrophys</title-short>
+    <title>Herzschrittmachertherapie + Elektrophysiologie (German)</title>
+    <title-short>Herzschrittmachertherapie Elektrophysiologie</title-short>
```

Comment:
The new `title-short` we have here comes directly from the official Springer list. However, for the titles here it might not be that important...